woocommerce/wc-admin
# Add transient notices

Fixes #806 fixes #1420 

Adds in support for transient notices so we can display "toast" style notifications.

### Accessibility


### Screenshots
<img width="851" alt="screen shot 2019-01-31 at 7 19 07 pm" src="https://user-images.githubusercontent.com/10561050/52050958-1e458600-258d-11e9-8aff-cc010076fc24.png">

### Detailed test instructions:

1.  Visit the Settings page `wp-admin/admin.php?page=wc-admin#/analytics/settings`.
2. Click on "Save changes."
3. Note the transient notice in the bottom right corner of the screen.
4. Using a screen reader, test the notice again and make sure that it's read aloud.

### Questions
* Should we keep using `wc-api` for this or is worth creating a separate store?
* Styling is taken directly from the Gutenberg component.  Should we make any changes here  @LevinMedia?
